template/js/netlify-cms/pages-preview.js

```diff
@@ -0,0 +1,5 @@
+import HomePreview from './preview/home-preview'
+
+window.CMS.registerPreviewStyle('/preview.css')
```

Comment:
Até acho que você vai precisar de um CSS adicional só para os previews mesmo, mas ele não foi criado ainda, certo?
Pra isto você precisa de um arquivo `template/scss/preview.scss`...

Reply:
Sim, é preciso registrar todo css usado pelo netlify, só incluir o css solto pelo cms não funciona. Por isso registrei esse preview.

Reply:
Sim porque eu to aplicando display block e none direto no javascript

Reply:
Uai mas então você pode remover este `window.CMS.registerPreviewStyle('/preview.css')`, não?
Já que o `preview.css` nem existe.